Makefile

```diff
@@ -22,12 +22,21 @@ build:
 	@-mkdir -p $(BUILD_FOLDER) 2> /dev/null
 	@go build $(BUILD_FLAGS) -o $(BUILD_FOLDER) ./...
 
+## mocks: generate mocks
+mocks:
+	@echo Generating mocks
+	@go install github.com/vektra/mockery/v2@latest
```

Comment:
We don't need to specify the version `@latest`  because it is already present in the `go.mod`, thanks to `ignite/internal/tools/tools.go`.

`go install github.com/vektra/mockery/v2` will install the exact version declared in the go.mod.

If you specify `@latest`, anyone may use a different `mockery` version (actually I did, my bad, but your change may fix that!)

This is pattern explained in the golang wiki https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module